### PR TITLE
resolve value changes issue

### PR DESCRIPTION
**Pull request by amitpatra91**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Style Update (CSS)
- [ ] Other... Please describe:

## What is the current behavior?
After change value, Updated value will display.

Resolves [738](https://github.com/mobi/goponents/issues/738)

## What is the new behavior?
After change value, Updated value will display.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts [hunk 1]
```diff
@@ -33,6 +33,14 @@ export class GoTimepickerComponent implements OnInit {
     this.selectedTime = this.changeTimeFormat(this.control.value);
   }
 
+  ngAfterViewInit(): void {
+    this.control.valueChanges.subscribe((value: any) => {
```
Comment (StevenUlmer) [inline projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts:37]: Should value be type `string | Date` here?

File: projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts [hunk 2]
```diff
@@ -33,6 +33,14 @@ export class GoTimepickerComponent implements OnInit {
...
+      if (value) {
```
Comment (StevenUlmer) [inline projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts:38]: Why are we checking if there is a value? If someone wants to programmatically null out the selectedTime should they be able to?

File: projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts [hunk 3]
```diff
@@ -33,6 +33,14 @@ export class GoTimepickerComponent implements OnInit {
...
+        this.selectedTime = this.changeTimeFormat(this.control.value);
```
Comment (StevenUlmer) [inline projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts:39]: Should we add a test for this?

**Review by StevenUlmer — CHANGES_REQUESTED** [review]

File: projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts (hunk 2 above)
Comment (amitpatra91) [reply projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts:38]: So, Can I remove the condition? Or I will compare the value with proper time format.

**Review by StevenUlmer — APPROVED** [review]
This makes sense to me

**Comment by amitpatra91** [comment]
Thanks.

**Pull request merged**